loading-indicator.android.ts

```diff
@@ -0,0 +1,49 @@
+import {OptionsI} from './loading-indicator.common';
+import * as application from 'application';
+
+declare var android: any;
+
+export class LoadingIndicator {
+  private _progressDialog: any;
+  
+  public show(options: OptionsI) {
+    let context = this._getContext();
+    if (context) {
+      if (options === undefined) options = {};
+      if (this._progressDialog === undefined) {
+        // Create
+        let indeterminate = true;
+        let cancelable = false;
+        if (options.android) {
+          if (typeof options.android.indeterminate !== 'undefined') indeterminate = options.android.indeterminate;
+          if (typeof options.android.cancelable !== 'undefined') cancelable = options.android.cancelable;
```

Comment:
Is there a reason for this more verbose check for the field being provided, compared to the simpler ones below? I think I like the simpler ones better, feels like it provides the same safety anyway.